tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-minversion = 3.5
+minversion = 3.7
```

Comment:
0041aad535836b5faf540205c8c68d5e42b52e19 drops Python 3.6 support but forgot to update `minversion` for `tox`.

Reply:
this is actually tox version (and I don't even remember why minversion was required here). But doesn't hurt anyway :)